test/count_set.hpp

```diff
@@ -54,7 +52,7 @@ struct count_set
     friend std::ostream &operator<<(std::ostream &os, const count_set& s)
     {
        os << "{";
-       BOOST_FOREACH(std::size_t const& value, s.m_values)
+       for(std::size_t const& value : s.m_values)
```

Comment:
This is non related but I just got it in too

Reply:
great! But there is a space missing after `for`

Reply:
:heavy_check_mark: thanks, fixed